regression/cbmc/r_w_ok_inconsistent_dead/test-global-no-cp.desc

```diff
@@ -0,0 +1,10 @@
+FUTURE
+main.c
+--function test_global_pointer --pointer-check --no-simplify --no-propagation
+^EXIT=0$
+^SIGNAL=0$
```

Comment:
:bulb: I think some more agressive checks (e.g verification successful?) would be good

Reply:
Added